### ISSUE TITLE
IDEA: graded descriptions

### ISSUE THREAD

**Issue by jtauber** [jtauber/labe-luxnos#5]
Unlike a normal text adventure, I think this game (system) could support various levels of description depending on language level.

So the first time you go to the harbour, it's just described as a harbour and there's a boat there. But depending on your vocabulary level (either pre-determined or increased throughout the game) things get described in more detail with new vocabulary words. The harbour is described. The boat is described. As are the sailors and what they are doing, etc.

**Comment by seumasjeltzz** [comment]
Depending on how you implement this, it adds whole layers of complexity. e.g., how dynamic that 'leveling' of language is, also on the complexity of the language used in writing. So my take would be: write for a basic level first, then build on that. Though, I suppose, if you wrote one version of a scene, you could then write more complex versions 'on top', like a reverse version of embedded readings.

**Comment by jtauber** [comment]
Yeah, I was just thinking of two or three levels of description just so we can increase the vocabulary coverage without creating new elements to the game. Maybe it's just something you can switch between (self-assessing).

It's also possible that you get to the more detailed descriptions just by looking at stuff in more detail.

Or perhaps later parts of the game that you can't reach at first have more elaborate descriptions.

For example, the "basic" version could say there's a στήλη but the richer description would let you read the the inscription. Or the "basic" version could say there's a ναύτης but the richer description would describe what he's wearing or what he's doing. 

Could definitely be written with the "easy" version first, a lot less descriptive than the average work of interactive fiction.

**Comment by seumasjeltzz** [comment]
I think self-selected would work fine, better than 'look in detail', because I think that one wants levels of description to be 'game neutral' (i.e. you should never need 'more language' to 'game' better). In terms of writing, yes, building from the ground up.
